Src/AutoFixture/DataAnnotations/MinAndMaxLengthAttributeRelay.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using AutoFixture.Kernel;
+
+namespace AutoFixture.DataAnnotations
+{
+    /// <summary>
+    /// Relays a request for a constrained string to a <see cref="ConstrainedStringRequest"/>.
+    /// </summary>
+    public class MinAndMaxLengthAttributeRelay : ISpecimenBuilder
+    {
+        private readonly IMemberTypeResolver memberTypeResolver;
+
+        /// <summary>
+        /// Creates a new instance of MinAndMaxLengthAttributeRelay
+        /// </summary>
+        /// <param name="memberTypeResolver">MemberTypeResolver to determine member type of requested object</param>
+        public MinAndMaxLengthAttributeRelay(IMemberTypeResolver memberTypeResolver)
```

Comment:
Please use property injection here instead of constructor injection. We have a good default for the `IMemberTypeResolver`, so no need for client to pass it.

Example:
https://github.com/AutoFixture/AutoFixture/blob/11066d77bcee17e4175679cfa0e9133957695671/Src/AutoFixture.NUnit3/AutoDataAttribute.cs#L27-L31